client/src/pages/reports/AttendeesTable.js

```diff
@@ -104,8 +108,43 @@ RadioButton.propTypes = {
   handleOnChange: PropTypes.func
 }
 
+const SensitiveCheckBox = props => {
+  const { person, disabled, handleOnChange } = props
+  return (
+    <Checkbox
+      // inline-block
+      className="sensitive-checkbox"
+      checked={person.sensitive}
+      disabled={disabled}
+      onChange={event => {
+        !disabled && handleOnChange(person, event.target.checked)
+        !disabled &&
+          event.target.checked &&
+          toast.info(
+            "You are restricting the visibility of the attendee to only authorized personnel. This will exclude that information from all reporting capabilities. Please do this only when necessary."
+          )
+      }}
+    >
+      {person.sensitive && <Label bsStyle="primary">Sensitive</Label>}
+    </Checkbox>
+  )
+}
+
+SensitiveCheckBox.propTypes = {
+  person: PropTypes.object,
+  disabled: PropTypes.bool,
+  handleOnChange: PropTypes.func
```

Comment:
I'm guessing that since you unconditionally deref them, both `person` and `handleOnChange` should be `.isRequired`.

Reply:
Yes, both are indeed required